launchdarkly/helper.go

```diff
@@ -0,0 +1,34 @@
+package launchdarkly
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"strings"
+)
+
+func parseCompositeID(id string) (p1 string, p2 string, err error) {
+	parts := strings.SplitN(id, ":", 2)
+	if len(parts) == 2 {
+		p1 = parts[0]
+		p2 = parts[1]
+	} else {
+		err = fmt.Errorf("error: Import composite ID requires two parts separated by colon, eg x:y")
+	}
+	return
+}
+
+type importFunc func(d *schema.ResourceData, meta interface{}) error
+
+func resourceImport(readMethod importFunc,d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	projectKey, resourceKey, err := parseCompositeID(d.Id())
+	if err != nil {
```

Comment:
The braces here are probably unneeded, right?

Reply:
You need those in Go, else it will not compile because of syntax error. I looked for it, but Go doesn't support ternary one-liners.

Reply:
You need those in Go, else it will not compile because of syntax error. I looked for it, but Go doesn't support ternary one-liners.

Reply:
True!